<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CodeEditor" name="editor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExitApp"/>
   </widget>
   <widget class="QMenu" name="menuEdit_O">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindReplace"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Выбор синтаксиса (для подсветки)</string>
     </property>
     <addaction name="actionSyntaxC2011"/>
     <addaction name="actionSyntaxCPP2014"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Выбор/Редактирование стиля подсветки</string>
     </property>
     <addaction name="actionChangeSyntaxStyle"/>
     <addaction name="actionOpenSyntaxStyleFromFile"/>
     <addaction name="actionDefaultSyntaxStyle"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionBackgroundColor"/>
    <addaction name="actionCurrentLineColor"/>
    <addaction name="separator"/>
    <addaction name="actionLineNumeration"/>
    <addaction name="actionInstrumentPanel"/>
    <addaction name="actionStatusString"/>
    <addaction name="actionSyntaxHighlight"/>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Формат</string>
    </property>
    <addaction name="actionWordWrap"/>
    <addaction name="actionFontChoose"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit_O"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="actionFindReplace"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/NewFile.png</normaloff>:/NewFile.png</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="toolTip">
    <string>Новый документ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/OpenFile.png</normaloff>:/OpenFile.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/undo.png</normaloff>:/undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/redo.png</normaloff>:/redo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/SaveFile.png</normaloff>:/SaveFile.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/SaveFile.png</normaloff>:/SaveFile.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/copy.png</normaloff>:/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/cut.png</normaloff>:/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/paste.png</normaloff>:/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/search.png</normaloff>:/search.png</iconset>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="actionFindReplace">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/searchAndReplace.png</normaloff>:/searchAndReplace.png</iconset>
   </property>
   <property name="text">
    <string>Найти и заменить</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перенос по словам</string>
   </property>
  </action>
  <action name="actionFontChoose">
   <property name="text">
    <string>Выбор шрифта</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="text">
    <string>Выбор цвета фона</string>
   </property>
  </action>
  <action name="actionCurrentLineColor">
   <property name="text">
    <string>Выбор цвета текущей строки</string>
   </property>
  </action>
  <action name="actionLineNumeration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Нумерация строк</string>
   </property>
  </action>
  <action name="actionInstrumentPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Панель инструментов</string>
   </property>
  </action>
  <action name="actionStatusString">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Строка состояния</string>
   </property>
  </action>
  <action name="actionSyntaxHighlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Подсветка синтаксиса</string>
   </property>
  </action>
  <action name="actionSyntaxC2011">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си (стандарт 2011)</string>
   </property>
  </action>
  <action name="actionSyntaxCPP2014">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Си++ (стандарт 2014)</string>
   </property>
  </action>
  <action name="actionChangeSyntaxStyle">
   <property name="text">
    <string>Изменить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменённый стиль</string>
   </property>
  </action>
  <action name="actionOpenSyntaxStyleFromFile">
   <property name="text">
    <string>Загрузка стиля из файла</string>
   </property>
  </action>
  <action name="actionDefaultSyntaxStyle">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
